### ISSUE TITLE
sudo required by assert_file_owner? 

### ISSUE THREAD

**Issue by flickerfly** [bats-core/bats-file#37]
When I run checks with assert_file_owner it seems to complain about sudo.

If I uncomment the assert_file_owner lines, this passes. 
```
@test "verify we can setup the log location" {
  INT_REPOSITORY_DIR="/tmp"
  LOG_DIR="$INT_REPOSITORY_DIR/logs/"
  rm -rf "$LOG_DIR"
  run log_setup
  assert_dir_exist "$LOG_DIR"
  assert_file_owner tcsuser "$LOG_DIR"  # This seems to be the problem line
}
```

If I leave either of the second to last line, I get an error like this.
```
 ✗ verify we can setup the bluray log location
   (in test file test/TCSstart_BLURAY_install.sh.bats, line 144)
     `assert_dir_exist "$LOG_DIR"' failed with status 127
   /opt/bats/lib/bats-file/src/file.bash: line 297: sudo: command not found
```

**Comment by leventyalcin** [comment]
I've fixed this issue in the PR #38. However, tests are failing at something unrelated. I'll come back at that and see if I make tests pass.

**Comment by leventyalcin** [comment]
Well, with the PR, I've solved 

* `sudo` placement bug
* regressions introduced with some other commits 
* easier local development environment for everyone. 

Hope it will be merged 👍
